string type
int64 point
---
string result